src/components/SearchResults.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import algoliaSearch from 'algoliasearch';
+import useSearchResults from './useSearchResults';
+import { SEARCH_PARAM_RESOURCE } from '../constants';
+
+const renderResults = (results: algoliaSearch.Response) => {
+  const { hits } = results;
+  if (hits && hits.length) {
+    const listItems = hits.map(({ charityname, objectID }) => {
+      if (charityname && objectID) {
+        return (
+          <li key={objectID}>
+            <Link
+              to={{
+                pathname: '/resource',
+                search: `?${SEARCH_PARAM_RESOURCE}=${objectID}`
+              }}
+            >
+              {charityname}
+            </Link>
+          </li>
+        );
+      }
+    });
+    return <ul>{listItems}</ul>;
+  }
+  return null;
```

Comment:
So if I'm reading this right, the hook returns `[]` for no hits and `null` if the query is still in progress. I haven't looked at this super closely, but just thinking maybe we want to show some loading indicator for the second case?

Reply:
The hook returns an `algoliaSearch.Response` object or null when not retrieved.

We will want to show a loading icon if it's still loading but I'm not going to worry about that for this PR. I imagine we'll want to use a material icon for that, but it may be wise to create a component for that to be used throughout the app.